packages/gdl-frontend/locale/en/en.json

```diff
@@ -0,0 +1,118 @@
+{
+  "Blog": "Blog",
+  "From": "From",
+  "Home": "Home",
+  "More": "More",
+  "Next": "Next",
+  "Sync": "Sync",
+  "from": "from",
+  "About": "About",
+  "Done!": "Done!",
+  "Level": "Level {level}",
+  "Share": "Share",
+  "Log in": "Log in",
+  "Oh no!": "Oh no!",
+  "Search": "Search",
+  "result": "{totalCount, number} {totalCount, plural, one {result for} other {results for}}",
```

Comment:
For å ikke komplisere oversettelses filen, kanskje bare sette `id: "results"` og `defaultMessage: "results"`? Slipper de som oversetter eller oss å håndtere dette ved hvert nytt språk.

Reply:
Da blir det `1 results for ...` og `5 results for ...` :)

Reply:
Må det, hvis ikke må vi deale med russisk :joy: